certified-connectors/Tesseron Invoice/apiDefinition.swagger.json

```diff
@@ -217,6 +217,408 @@
+                  "properties": {
+                    "ShowActivityRecordings": {
+                      "type": "boolean",
+                      "description": "Specify whether you want to receive matching activity recordings.",
+                      "title": "Show Activity Recordings?",
+                      "enum": [
+                        true,
+                        false
+                      ],
+                      "default": false
+                    },
+                    "ShowPositionNotes": {
+                      "type": "boolean",
+                      "description": "Specify whether you want to receive matching position notes.",
+                      "title": "Show Position Notes?",
+                      "enum": [
+                        true,
+                        false
+                      ],
+                      "default": false
+                    },
+                    "ShowAssignmentPositions": {
+                      "type": "boolean",
+                      "description": "Specify whether you want to receive single assignment positions.",
+                      "title": "Show Assignment Positions?",
+                      "enum": [
+                        true,
+                        false
+                      ],
+                      "default": true
+                    },
+                    "SummarizePositionsIntoAssignment": {
+                      "type": "boolean",
+                      "description": "Specify whether you want to receive single assignment positions.",
+                      "title": "Summarize into Assignment?",
+                      "enum": [
+                        true,
+                        false
+                      ],
+                      "default": false
+                    },
+                    "AssignmentStatusId": {
+                      "type": "integer",
+                      "description": "Specify your status.",
+                      "title": "Assignment Status"
+                    },
+                    "PositionNoteStatus": {
+                      "type": "integer",
+                      "description": "Specify your position status.",
+                      "title": "Assignment Position Status"
+                    },
+                    "DateTimeFrom": {
+                      "type": "string",
+                      "description": "Specify your starting date.",
+                      "title": "Date from"
+                    },
+                    "DateTimeTo": {
+                      "type": "integer",
+                      "description": "Specify your end date.",
+                      "title": "Date to"
+                    },
+                    "IsInvoice": {
+                      "type": "boolean",
+                      "description": "Specify whether you want to receive a specific billing type.",
+                      "title": "Is invoice?",
+                      "enum": [
+                        null,
```

Comment:
The type of the enum value is 'Null' but it should be 'Boolean' as specified in the schema. ValuePath: /paths/~1GetServiceAssignmentsDispatcher/post/parameters/0/schema/properties/AdditionalSearchData/properties/IsInvoice/enum

Reply:
option null deleted